pkg/operator/status.go

```diff
@@ -250,15 +263,24 @@ func (c *Controller) computeStatusConditions(tuned *tunedv1.Tuned, conditions []
 
 				copyAvailableCondition()
 			}
-			reportDegraded = anyProfileDegraded(profileList)
+			reportProfileDegraded = anyProfileDegraded(profileList)
+			reportProfileProgressing = anyProfileProgressing(profileList)
 		} else {
-			// Profile supplied, check if Degraded should be reported based on its status conditions
-			reportDegraded = profileDegraded(profile)
+			// Profile supplied, check if ProfileDegraded should be reported based on its status conditions
+			reportProfileDegraded = profileDegraded(profile) || !profileApplied(profile)
+			// If the profile is not applied, and not degraded, we should report that the operator is progressing.
+			reportProfileProgressing = !profileApplied(profile) && !profileDegraded(profile)
 		}
-		if reportDegraded {
+		if reportProfileDegraded {
 			klog.Infof("at least one Profile application failing")
 			availableCondition.Reason = "ProfileDegraded"
-			availableCondition.Message = fmt.Sprintf("At least one Profile application failing")
+			availableCondition.Message = "At least one Profile application failing"
+		}
+
+		if reportProfileProgressing {
+			progressingCondition.Status = configv1.ConditionTrue
+			progressingCondition.Reason = "ProfileProgressing"
+			progressingCondition.Message = "Waiting for at least one Profile to be applied"
```

Comment:
This message is ambiguous and as an admin/user I'd be confused.  Assume you have a 6 node cluster with 6 profiles already applied.  You create a config for NTO to change a profile on one node.  The user willt then see this message in spite of 5 profiles already applied.  Possible solution:
  - rephrase the message
  - do not report progressing for `co/node-tuning` at all
  - improve the code to report how many profiles are applied currently and for how many we are waiting to be applied

I'll leave the decision up to you.  In general, I'm having second thoughts about reporting this via `co` at all, but on the other way, I like the third suggestion best **right now** :)  This needs a bit more thought or asking/looking around how other operators are dealing with this when deploying configuration to their operands.